crypto/ube/fork_detect.c

```diff
@@ -144,15 +139,15 @@ uint64_t CRYPTO_get_fork_generation(void) {
     if (current_generation == 0) {
       current_generation = 1;
     }
-    *generation_ptr = current_generation;
+    fork_generation = current_generation;
   }
   CRYPTO_STATIC_MUTEX_unlock_write(lock);
 
   return current_generation;
 }
 
 void CRYPTO_fork_detect_ignore_madv_wipeonfork_for_testing(void) {
```

Comment:
For consistency, and because it is private technically, ca we rename this to match the `FOR_TESTING` uppercase pattern?